docs/content/3.forms/11.radio-group.md

```diff
@@ -64,7 +78,7 @@ props:
 
 ### `label`
 
-Use the `#label` slot to override the content of the label.
+Use the `#label` slot to override the label of each options.
```

Comment:
Typo: 'each options' -> 'each option' or 'each of the options'

Reply:
```suggestion
Use the `#label` slot to override the label of each option.
```